internal/worker/rundynamic.go

```diff
@@ -120,6 +124,37 @@ func retrieveExecutionLog(ctx context.Context, sb sandbox.Sandbox) (string, erro
 	return string(processedLog), nil
 }
 
+func addSSHKeysToSandbox(sb sandbox.Sandbox, ctx context.Context) error {
+	generatedPrivateKey, err := rsa.GenerateKey(rand.Reader, 2048)
+	generatedPublicKey := generatedPrivateKey.PublicKey
+	if err != nil {
+		return err
+	}
+
+	os.Mkdir("temp_ssh_dir", 0750)
```

Comment:
prefer `tempdir, err := os.MkdirTemp(".", "temp_ssh_dir")` so the dirname is in a variable.

Then you can use `filepath.Join` in expressions below to create the files in the temp directory

Reply:
Also defer a removal of the `tempdir`

Reply:
Done.